lib/protobuf/generators/file_generator.rb

```diff
@@ -125,7 +125,7 @@ def print_import_requires
         header "Imports"
 
         descriptor.dependency.each do |dependency|
-          print_require(convert_filename(dependency))
+          print_require(convert_filename(dependency), ENV['REQUIRE_RELATIVE'] == 'true')
```

Comment:
We've been prefacing env vars with `PB_`. Would you mind doing that here? `PB_REQUIRE_RELATIVE`?